### PR TITLE
get_customs outputs array whit keys 'label' and 'value'

### PR DESCRIPTION
**Pull request by kas-cor**
```php
<?php
$array = get_customs('array_setting', true);
foreach ($array as $name) :
    echo $name['label'] . " - " . $name['value'];
endforeach;

// Old construct is worked
$array = get_customs('array_setting');
foreach ($array as $name) :
    echo $name;
endforeach;
```

### CODE REVIEW

**Comment by leocaseiro** [comment]
Hi @kas-cor, thank you for sending this PR. It seems pretty good.

I can't believe I've never thought making that. LOL

I'll make some tests this week and will release a new version soon.
Please remind me if I take longer than a week or so. Doing lot's of projects atm. 
Thanks heaps.

File: custom-options-plus.php [hunk 1]
```diff
@@ -347,13 +347,13 @@ function get_custom( $name ) {
 }
 
 // Get your array options
-function get_customs( $name ) {
+function get_customs( $name, $label = false) {
```
Comment (leocaseiro) [inline custom-options-plus.php]: I'd like to have a [space between parentheses]((https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage).

File: custom-options-plus.php [hunk 2]
```diff
@@ -347,13 +347,13 @@ function get_custom( $name ) {
...
 	global $wpdb, $COP_TABLE;
 	if ( '' != $name ) :
-		$list  = $wpdb->get_results( $wpdb->prepare( "SELECT value FROM $COP_TABLE WHERE name = %s ", $name ), ARRAY_A );
+		$list  = $wpdb->get_results( $wpdb->prepare( "SELECT label,value FROM $COP_TABLE WHERE name = %s ", $name ), ARRAY_A );
```
Comment (leocaseiro) [inline custom-options-plus.php]: Please put a space between label and value, thanks

File: custom-options-plus.php [hunk 3]
```diff
@@ -347,13 +347,13 @@ function get_custom( $name ) {
...
 		$array = array();
 		foreach ( $list as $key => $name ) :
-			$array[] = $name['value'];
+			$array[] = $label ? ['label' => $name['label'], 'value' => $name['value'] : $name['value'];
```
Comment (leocaseiro) [inline custom-options-plus.php]: It seems there's a parse error here: ` ['label'`

```php
echo $array[] = $label ? ['label' => $name['label'], 'value' => $name['value'] : $name['value'];
```

File: custom-options-plus.php (hunk 1 above)
Comment (kas-cor) [inline custom-options-plus.php]: ok

**Comment by leocaseiro** [comment]
Closed this PR in regards to #20

**Pull request closed without merging**